core/DataAccess/Model.php

```diff
@@ -205,20 +205,14 @@ public function deleteArchiveIds($numericTable, $blobTable, $idsToDelete)
         return $deletedRows;
     }
 
-    public function getArchiveIdAndVisits($numericTable, $idSite, $period, $dateStartIso, $dateEndIso, $minDatetimeIsoArchiveProcessedUTC, $doneFlags, $doneFlagValues)
+    public function getArchiveIdAndVisits($numericTable, $idSite, $period, $dateStartIso, $dateEndIso, $doneFlags, $doneFlagValues)
     {
         $bindSQL = array($idSite,
             $dateStartIso,
             $dateEndIso,
             $period,
         );
 
-        $timeStampWhere = '';
```

Comment:
@diosmosis do we maybe need an update script to get rid of all temporary archives? Maybe some temporary archives would need to be converted to a completed archive? (not sure if that's an issue but wondering if it's possible that we were sometimes reading a temporary archive because for some reason no completed archive was created)

Reply:
This code should still look for temporary archives in addition to done archives (whichever is latest), so it shouldn't be necessary to do it now. I could add the update though.